test/check-application

```diff
@@ -630,5 +630,53 @@ class TestApplication(testlib.MachineCase):
         ).execute()
 
 
+    def testResubmitCert(self):
+        b = self.browser
+        m = self.machine
+
+        cert_name = "Server-Cert"
+        db_path = "/etc/pki/nssdb"
+        dns_name = "example.com"
+        m.execute("selfsign-getcert request -d {0} -n {1} -D {2}".format(db_path, cert_name, dns_name))
+        self.addCleanup(self.cleanupCertificate, cert_name, db_path)
+
+        self.login_and_go("/certificates")
+        # Verify expected heading
+        b.wait_in_text(".ct-table-header h3", "Certificates")
+
+        b.wait_in_text("#certificate-0-name", cert_name)
+        b.click("button#expandable-toggle0")
+
+        old_validity = m.execute("selfsign-getcert list -d %s -n %s | grep 'issued:'" % (db_path, cert_name)).strip()
+
+        b.wait_not_present("#certificate-0-general-principal")
+        b.wait_in_text("#certificate-0-general-dns", dns_name)
+
+        # Resubmit certificate with different signing request parameters
+        b.click("#certificate-0-action-kebab button")
+        b.click("#certificate-0-resubmit a")
+        b.wait_in_text(".pf-c-modal-box .pf-c-modal-box__header .pf-c-modal-box__title", "Resubmit Certificate")
+
+        # Change signing request propertes for resubmit
+        subject_name="CN=TEST_HOSTAME"
+        principal_name = "TEST_HOSTNAME@REDHAT.COM"
+        b.set_input_text("#subject-name", subject_name)
+        b.set_input_text("#principal-name", principal_name)
+        b.set_input_text("#dns-name", "") # Remove DNS name
+
+        b.click(".pf-c-modal-box__footer button:contains(Resubmit)")
+        b.wait_not_present(".pf-c-modal-box")
+        time.sleep(4) # Certificate enrollment takes some time
```

Comment:
Alright, but no I realized that I have to add another time.sleep(1) just before clicking "resubmit" button, because the success of resubmition is tested by checking if expiration date has changed, and since expiration date is shown in seconds, we have to make sure that resubmition takes at least one second after the original certificate was created.

Reply:
That's fair -- please add the sleep(2) (just to make sure to not walk into rounding errors) with a comment that points this out. Thanks!